app/helpers/publishers_helper.rb

```diff
@@ -256,4 +256,10 @@ def publisher_filter_public_file_content(publisher, file_content)
       ""
     end
   end
+
+  def name_from_email(email)
+    return "Publisher" unless email.is_a?(String)
+
+    email.split("@")[0].capitalize
```

Comment:
IMO this is going to look funny more times than not. e.g. if Alice Blogger at BlogLandLLC had an email ablogger@blogland.world it would show as `Hi, ablogger`

i think we can correct this in a followup #270